polkadot-host-spec/c02-state.tm

```diff
@@ -396,6 +396,42 @@
     </algorithmic>
   </algorithm>
 
+  <subsection|Child Tries><label|sect-child-trie-structure>
+
+  Child tries are trie structures as defined Section
```

Comment:
so you say
> [child tries] are saved as subcomponents within the Polkadot state trie.
1. what does subcomponent means?
then you say:
> Polkadot state trie only links to an individual child trie by its corresponding Merkle proof
2. What does links mean in the context of trie mathematically?
so because it is radix 16 trie you only need to tell me what is the (key,value) for the child trie then I can use radix 16 insertion to insert it in the trie
so my guess is that you mean value = hash of the child trie?
then what is the key?
3. as an implementor suppose 1. you embed the child trie into the state trie you are still compling with the spec because you are going to compute the correct merkle for the state trie, right?
so if 3 is correct, then why do we need to even mention the child trie, if it is just an optimization and doesn't change the final merkle hash value?